
VERSION ""


NS_ : 
        NS_DESC_
        CM_
        BA_DEF_
        BA_
        VAL_
        CAT_DEF_
        CAT_
        FILTER
        BA_DEF_DEF_
        EV_DATA_
        ENVVAR_DATA_
        SGTYPE_
        SGTYPE_VAL_
        BA_DEF_SGTYPE_
        BA_SGTYPE_
        SIG_TYPE_REF_
        VAL_TABLE_
        SIG_GROUP_
        SIG_VALTYPE_
        SIGTYPE_VALTYPE_
        BO_TX_BU_
        BA_DEF_REL_
        BA_REL_
        BA_DEF_DEF_REL_
        BU_SG_REL_
        BU_EV_REL_
        BU_BO_REL_
        SG_MUL_VAL_

BS_:

BU_: FC PI


BO_ 454 PedalSensors: 6 FC
    SG_ Apps1 : 0|12@1+ (1,0) [0|3.3] "V" PI
    SG_ Apps2 : 12|12@1+ (1,0) [0|3.3] "V" PI
    SG_ Bpps1 : 24|12@1+ (1,0) [0|3.3] "V" PI
    SG_ Bpps2 : 36|12@1+ (1,0) [0|3.3] "V" PI

BO_ 898 Wheels: 2 FC
    SG_ WheelSpeedRightA : 0|1@1+ (1,0) [0|1] "" PI
    SG_ WheelSpeedRightB : 1|1@1+ (1,0) [0|1] "" PI
    SG_ WheelSpeedLeftA : 2|1@1+ (1,0) [0|1] "" PI
    SG_ WheelSpeedLeftB : 3|1@1+ (1,0) [0|1] "" PI
    SG_ SteeringAngle : 4|12@1+ (1.0000001,0) [0|3.3] "V" PI

BO_ 983 VehicleInfo: 4 FC
    SG_ HvilFeedback : 0|12@1+ (1.0000001,0) [0|3.3] "V" PI
    SG_ StartButton : 24|1@1+ (1,0) [0|0] "" PI

BO_ 1032 Suspension: 3 FC
    SG_ SuspensionTravel1 : 0|12@1+ (1.0000001,0) [0|3.3] "V" PI
    SG_ SuspensionTravel2 : 12|12@1+ (1.0000001,0) [0|3.3] "V" PI

BO_ 634 RequiredRxMessage: 1 PI
    SG_ DoNotUseMe : 0|1@1+ (1,0) [0|1] "" FC
    SG_ ThisWillBeRemovedSoon : 1|1@1+ (1,0) [0|1] "" FC

